### PR TITLE
plan: estimate ndv as pseudo when its value is zero

### PR DESCRIPTION
**Pull request by hanfei1991**
Before we analyze table, the ndv is always 0. We should take it as count * distinct factor when we do estimation.

### CODE REVIEW

File: plan/cbo_test.go [hunk 1]
```diff
@@ -34,6 +35,32 @@ var _ = Suite(&testAnalyzeSuite{})
 type testAnalyzeSuite struct {
 }
 
+// this function tests the plan with stats that only have count info
```
Comment (lamxTyler) [inline plan/cbo_test.go]: this -> This, add dot at the end.

Comment (shenli) [reply plan/cbo_test.go]: Could you add some comments to explain why tables does not have stats info?

Comment (hanfei1991) [reply plan/cbo_test.go]: @shenli  I have closed auto analyze

**Comment by shenli** [comment]
LGTM

**Pull request merged**